manual/de/Help_ch_Tools_Assistants.xml

```diff
@@ -4188,136 +4188,191 @@
   </sect1>
 
   <sect1 id="trans-sched-slr">
-    <title>Since Last Run Assistant</title>
-
-    <para>The <guilabel>Since Last Run</guilabel> assistant is run automatically when &app; is started. It is
-      used to enter into the register any transactions that are due to be automatically entered (see
-      <xref linkend="trans-sched"></xref>). The run on &app; start can be altered in the
-      <link linkend="prefs-sched">Scheduled Transactions</link> tab of the <guilabel>&appname;
-      Preferences</guilabel>. To run the <guilabel>Since Last Run</guilabel> assistant manually, go
-      to
+    <title>Der <quote>Seit letztem Aufruf</quote> Dialog</title>
+
+    <para>Der <guilabel>Seit letztem Aufruf</guilabel> Dialog wird automatisch ausgeführt, während &app;
+      startet. Er dient der Eintragung von Buchungen in das Kontobuch, die automatisch erfaßt
+      werden sollen (siehe <xref linkend="trans-sched" />). Die Ausführung während des Starts kann
+      in dem <xref linkend="prefs-sched" /> Reiter in den <guilabel>&appname;
+      Eigenschaften</guilabel> angepasst werden. Möchten Sie den Assistenten manuell auszuführen,
+      gehen Sie im Menu auf
       <menuchoice>
-        <guilabel>Actions</guilabel><guisubmenu>Scheduled Transactions</guisubmenu>
-        <guimenuitem>Since Last Run...</guimenuitem>
+        <guilabel>A<accel>k</accel>tionen</guilabel><guisubmenu>Term<accel>i</accel>nierte
+        Buchungen</guisubmenu> <guimenuitem>Seit <accel>l</accel>etztem Aufruf…</guimenuitem>
```

Comment:
Haben wir da jetzt nicht so eine tolle Entität für?

Reply:
Thanks. You directed me to some errors with the entities. See PR #265 .

Reply:
However, I have the impression that the `<menuchoice>` must also be defined as an entity before use.
I have yet to verify that.